provisioning/tasks/init-debian.yml

```diff
@@ -20,19 +20,24 @@
   apt_repository: repo='ppa:ondrej/apache2'
   when: ansible_distribution_release == "precise" and ansible_distribution == "Ubuntu"
 
-- name: Add repository for PHP 5.5.
-  apt_repository: repo='ppa:ondrej/php5'
-  when: php_version == "5.5" and ansible_distribution == "Ubuntu" and ansible_distribution_version == "12.04"
+- name: Add repository for PHP on Ubuntu.
+  apt_repository: repo='ppa:ondrej/php'
+  when: ansible_distribution == "Ubuntu" and ansible_distribution_version != "16.04"
```

Comment:
I tested it briefly locally but the build failed on 16.04 while 14.04 worked out of the box. I hope to get a second look at it this weekend.

That would be awesome. Being able to switch versions without destroying would be quite useful!